### ISSUE TITLE
Displayed multi-pv in scid-vs-pc v4.23

### ISSUE THREAD

**Issue by BrennusJ** [peterosterlund2/texel#4]
Hello,

The multi-pv display seems impossible in scid-vs-pc (regardless the version 1.08 1.09 ...)
i sent à screen capture where it shown the display for SF16 set with multipv at 3 lines and same for Texel
from both side the setting is 3 lines and SF16 (or other engines) is displayed 3 line but not Texel
i join the engine dialog log from scid-vs-pc

Is it a bug ?
What do you need more ? 
How i can help (by testing) ?

Regards

[texel-log-scidvs-pc.log](https://github.com/peterosterlund2/texel/files/12643296/texel-log-scidvs-pc.log)
![Capture d’écran_2023-09-17_18-54-26](https://github.com/peterosterlund2/texel/assets/3634605/7e9b84fc-20b5-407d-b52d-0d143b525d57)

**Comment by peterosterlund2** [comment]
Hi,

Texel only supports MultiPV in analysis mode and it looks like you are playing a game. This can probably be fixed by changing 

https://github.com/peterosterlund2/texel/blob/e5aca62d0cd165fc5b0d742112d4322d097fba9a/app/texel/enginecontrol.cpp#L478

to:

int maxPV = UciParams::multiPV->getIntPar();

I don't know if this is a bug because the UCI specification is unclear on the details of MultiPV. At the time I wrote that code I did not think it would make sense to use MultiPV when playing games, as it would just make the engine weaker. However, if Stockfish allows that I suppose it does not cause problems with typically used GUIs, so maybe I should change it in Texel too.
